src/ethLock.js

```diff
@@ -5,7 +5,7 @@ import jswallet from 'ethereumjs-wallet';
 import fs from 'fs';
 
 const { toBN, hexToNumber } = utils;
-export const LOCKDROP_JSON = JSON.parse(fs.readFileSync('./eth/build/contracts/Lockdrop.json').toString());
+//export const LOCKDROP_JSON = JSON.parse(fs.readFileSync('./eth/build/contracts/Lockdrop.json').toString());
```

Comment:
I had to comment this out because it wasn't running locally with this line in.

Reply:
I'll see if I can fix this before merging.

Reply:
Fixed?

Reply:
How do I get the build files?